test/chpldoc/globals/globalOrdering.good

```diff
@@ -1,11 +1,11 @@
 Module: globalOrdering
    config var goop: bool
       I should be first 
-   config const resorted
-      I should be second 
    param after
       I should be third 
    var fluctuates: int
       I should be fourth 
+   config const resorted
+      I should be second 
```

Comment:
With source-code based ordering, these comments are confusing and incorrect.  Would you mind updating them to be correct in either case?  i.e "under functional ordering, I should be second, but under source ordering I remained fourth", or something to that effect.